### ISSUE TITLE
Import the rest of the meetings

### ISSUE THREAD

**Issue by patricoferris** [ocaml/ood#30]
As part of #23 we need to bring in all of the data associated with meetings and events: 

 - Meetings: https://github.com/ocaml/ocaml.org/blob/master/site/meetings/index.md

In fact I think "meetings" and "events" are the same and here we have events. The problem is that the individual meeting pages (where they exist) tend to be very different across the board. For example the OCaml Workshop pages have a fairly straightforward structure in that they contain papers, presentations, organising committee etc. they can be found 
here https://github.com/ocaml/ocaml.org/tree/master/site/meetings/ocaml

Other meetings are simply a link to a meetup page. They don't necessarily need all of the data that the OUD ones do. In addition to this a lot of that data can be derived simply from the tags and years of the `papers.yml` entries and the `videos.yml` entries by filtering for the `ocaml-workshop` and the year `2020` for instance.

The meetings probably have enough additional content to be represented as a collection in a folder rather than as a single yaml file as they are currently. 

Perhaps the OUD entries just need a relation to papers and videos to make that link explicit? @tmattio any thoughts on the best way to approach this? Maybe we keep and `events.yml` for tracking meetups and the like that don't need their own individual page on the front-end side and then generate a `meetings/en/{meeting}.md` collection for primarily the OCaml workshop pages?

**Comment by tmattio** [comment]
What are the different types of meetings we have?

IIUC, there is the OCaml Workshop that are a bit specific because they are extremely defailed and could deserve a category on their own. Then there are the meetups, maybe also the OCaml Platform meetings (e.g. ocamlformat meeting, dune meetings, etc.), where the structure and level of detail should be relatively uniform. Am I missing something?

I would suggest dividing all of these into multiple categories. The information available seems to be uniform amongst each category of meetings, so we can model the types accordingly.

If there is a need to aggregate all of the meetings in one place in the frontend, the logic can be implemented there easily.

What do you think?

**Comment by patricoferris** [comment]
Yep that sounds very reasonable and a good idea. For now this primarily means having Workshop ones and then some more random meetings as we don't track Platform specific ones for now. Pushing the aggregation through to the front-end sounds like a scalable solution 👍 Thanks, I'll work on this :))
